### ISSUE TITLE
Exhibition with content "null"

### ISSUE THREAD

**Issue by LianWintermans** [antoinecourtin/VanGoghDataproject#3]
The exhibition in F822 has name/actor/place with content or label value "null".
See for example: https://github.com/antoinecourtin/VanGoghDataproject/blob/4fb059b0a71325accaa5e34217070059fbb506cd/F822.json#L281

And those values are literally included in https://preprod.vangoghworldwide.org/artwork/F822.

**Comment by LianWintermans** [comment]
By the way: I also noticed that in this exhibition [issue 2](https://github.com/antoinecourtin/VanGoghDataproject/issues/2) still is present (but that seems to be the only instance).

**Comment by antoinecourtin** [comment]
yes, we do not have any documentary information on this work because it is on deposit (see my last question by mail on how to differentiate in your model the owner of the work (the Musée d'Orsay) and the place of deposit of the work (Musée des Beaux-Arts de Lille)
